apps/api/src/app/auth/services/auth.service.ts

```diff
@@ -259,12 +266,16 @@ export class AuthService {
 
   @Instrument()
   async validateUser(payload: IJwtPayload): Promise<UserEntity> {
-    const user = await this.getUser({ _id: payload._id });
-    if (!user) throw new UnauthorizedException('User not found');
+    // We run these in parallel to speed up the query time
+    const userPromise = this.getUser({ _id: payload._id });
+    const isMemberPromise = payload.organizationId
```

Comment:
wondering when there is no `organizationId`? after the registration?

Reply:
Yep exactly, before the user creates the actual organization